src/styles/utils/_mixins.scss

```
@@ -1,6 +1,38 @@
 @mixin hover($_property, $_toValue) {
-  transition: #{$_property} 0.3s;
+  transition: #{$_property} 1s;
   &:hover {
     #{$_property}: $_toValue;
   }
 }
+
+@mixin Phone {
+
+  @media (max-width:640px) {
+    @content;
+  }
+  
+}
+
+@mixin Tablet {
+
+  @media (min-width:641px) and (max-width:1023px)  {
+    @content;
+  }
+  
+}
+
+@mixin Full {
+
+  @media (min-width:1441px)  {
+    @content;
+  }
+  
+}
+
+@mixin Desktop {
+
+  @media (min-width:1024px)  {
+    @content;
+  }
```

```suggestion
@mixin Desktop {

  @media (max-width:1440px)  {
    @content;
  }
```
И расположи его перед Full